docs/OperatorKernels.md

```diff
@@ -6,7 +6,6 @@ Do not modify directly.*
 
 - [CPUExecutionProvider](#cpuexecutionprovider)
 - [CUDAExecutionProvider](#cudaexecutionprovider)
-- [DmlExecutionProvider](#dmlexecutionprovider)
```

Comment:
DmlExecutionProvider section shall not be removed. You can copy the docs from artifacts from a doc pipeline.